docs/current_docs/quickstart/build.mdx

```diff
@@ -0,0 +1,85 @@
+---
+slug: /quickstart/build
+title: "Build the application"
+---
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Quickstart
+
+## Build the application
+
+The `build` stage of the pipeline creates a container image with a production build of the application. It also depends on the `build-env` stage. Let's look at its Dagger Function next.
+
+### Inspect the Dagger Function
+
+<Tabs groupId="language">
+<TabItem value="Go">
+
+```go file=./snippets/build/go/main.go
+```
+
+</TabItem>
+<TabItem value="Python">
+
+```python file=./snippets/build/python/__init__.py
+```
+
+</TabItem>
+<TabItem value="TypeScript">
+
+```typescript file=./snippets/build/typescript/index.ts
+```
+
+</TabItem>
+</Tabs>
+
+
+This Dagger Function performs a multi-stage build, as follows:
+
+- It starts with the build environment `Container` and executes the `npm run build` command in the container. This creates a compressed, production-ready build of the application and places the result in a `dist/` directory in the container filesystem. This directory is stored and referenced as Dagger's special `Directory` type.
+- It initializes a new `nginx:1.25-alpine` image as a second `Container` and copies the directory to the NGINX Web server root directory in the container. It also exposes container port 80 (the default NGINX port).
+
+The result of this Dagger Function is a `Container` representing the final production build of the application. This is an `nginx:1.25-alpine` container image with an NGINX Web server ready to host and serve the built application.
+
+### Call the Dagger Function
+
+Call the Dagger Function as below:
+
+```shell
+dagger call \
+  build --source=.
+```
+
+Here's what you should see:
+
+![Final container](/img/current_docs/quickstart/build.gif)
```

Comment:
I owe you a function to generate this. Sorry I didn't get to it, I didn't forget!